cynthion/python/src/gateware/platform/cynthion_r0_6.py

```diff
@@ -156,10 +156,10 @@ class CynthionPlatformRev0D6(CynthionPlatform):
         ),
 
         # User I/O connections.
-        Resource("user_pmod", 0, Pins("C9 C8 D11 C12 B8 D8 D9 C10", dir="io"), Attrs(IO_TYPE="LVCMOS33")),
-        Resource("user_pmod", 1, Pins("B4 B5 B6 B7 C5 A5 A6 A7", dir="io"), Attrs(IO_TYPE="LVCMOS33")),
+        Resource("user_pmod", 0, Pins("1 2 3 4 7 8 9 10", conn=("pmod", 0), dir="io"), Attrs(IO_TYPE="LVCMOS33")),
+        Resource("user_pmod", 1, Pins("1 2 3 4 7 8 9 10", conn=("pmod", 1), dir="io"), Attrs(IO_TYPE="LVCMOS33")),
         Resource("user_mezzanine", 0,
-                Pins("B9 A9 B10 A10 B11 D14 C14 F14 E14 G13 G12 C16 C15 B16 B15 A14 B13 A13 D13 A12 B12 A11", dir="io"),
+                Pins("3 4 5 6 7 8 9 10 11 12 13 18 19 20 21 22 23 24 25 26 27 28", dir="io"),
```

Comment:
Isn't a `conn=("mezzanine", 0)` argument needed?

Reply:
That is indeed the case! 🙈 Tx @mndza, I've updated the PR !